app/static/src/app/components/options/GraphSettings.vue

```diff
@@ -8,6 +8,14 @@
         <input type="checkbox" class="form-check-input" style="vertical-align:bottom;" v-model="logScaleYAxis">
       </div>
     </div>
+    <div id="lock-axes" class="row my-2">
```

Comment:
An exciting thing happens if you interact with the log scale toggle too.

1. Start with your example showing exponential growth
2. Lock the y axis
3. Select "log scale"

Now the y axis will go to something like 10^61800 and the lines are all along the bottom.

It's not the end of the world - unlocking the axis returns to sensible values, and then you can relock the axis and it behaves well in log mode. But this does not tolerate well moving between those modes

Reply:
fixed!